R/constructors.R

```diff
@@ -166,23 +166,48 @@ new_qual_param <- function(type = c("character", "logical"), values,
 
 #' @export
 print.quant_param <- function(x, digits = 3, ...) {
-  if (!is.null(x$label)) {
-    cat(x$label, " (quantitative)\n")
-  } else
-    cat("Quantitative Parameter\n")
-  if (!is.null(x$trans)) {
-    print(eval(x$trans))
-    cat("Range (transformed scale): ")
-  } else
-    cat("Range: ")
+  cat_header <- function(x) {
```

Comment:
Not sure if nesting functions like this is "bad style." I reviewed the style guide before contributing and didn't see anything suggesting it was. Since I typically work in languages where this sort of thing is fine (javascript, kotlin), I figured I'd give it a go. Let me know if you want these functions moved elsewhere.

Reply:
It's fine with me so long as we won't need it anywhere else